db/seeds/development/timeline_events.seeds.rb

```diff
@@ -67,14 +67,13 @@
         )
 
         # Set passed_at if all grades are over the pass grade.
-        if reviewed_submission
-             .timeline_event_grades
-             .includes(:evaluation_criterion)
-             .all? do |grade|
-               grade.grade >= grade.evaluation_criterion.pass_grade
-             end
+        random_passed_boolean = [true, false].sample
+        if reviewed_submission && random_passed_boolean
```

Comment:
Why don't we create a few failed submissions for targets separately?

Reply:
I was just trying to stick to the same code structure as before. Create submissions and fail or pass them randomly. Anything specific you are looking for by this change?

Reply:
On a call we had decided it would be okay to stick to the same structure.